### PR TITLE
Redesign model selection in project setup

### PR DESCRIPTION
**Pull request by J535D165**
image

### CODE REVIEW

**Comment by jteijema** [comment]
We'd recommend the following models:

TF-IDF
OneHot

Naive Bayes
Logistic Regression
SVM
Random Forest

---

Based on prelim results, we'd recommend the following combinations:

nb + tfidf
logistic + onehot

**Always Good**: A classic combination that has proven to work well in the ASReview context, as shown in many simulations. It especially handles the starting phase of the systematic review well, being able to handle little amounts of data.
**One Word**:  A model that excels in finding specific words, providing good performance in finding the last remaining relevant documents.

---

For the advanced models we'd suggest the following:

labse + logistic (multilingual)
sbert + xgboost (new implementation of sbert that handles token length limits)
wide-doc2vec + dynamic_nn

**Multi Language**: A multilingual feature extractor for systematic review datasets in multiple languages (LaBSE x Logistic Regression).
**Context**: A powerful pretrained model based on the transformer infrastructure used in the latest AI models. Combined with a proven implementation of a gradient boosting classifier (SBert x XGBoost).
**Neural**: A classifier and feature extractor combination both based on neural network architectures. Long training time and needs more data, but unrivaled performance in the final screening stages (Wide-Doc2Vec x Dynamic Neural Network).
